ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParsePayload.java

```diff
@@ -140,18 +140,14 @@ private ParsePayload(ValueProvider<String> schemasLocation) {
 
       // If no "document_version" attribute was parsed from the URI, this element must be from the
       // /submit/telemetry endpoint and we now need to grab version from the payload.
-      if (!attributes.containsKey("document_version")) {
```

Comment:
This logic is refactored to a public function here so that we can reuse it for AET.